core/src/main/java/com/linecorp/armeria/client/proxy/ProxyConfig.java

```diff
@@ -16,83 +16,86 @@
 
 package com.linecorp.armeria.client.proxy;
 
-import static com.linecorp.armeria.client.proxy.NoopProxyConfig.NOOP_PROXY_CONFIG;
+import static com.linecorp.armeria.client.proxy.DisabledProxyConfig.DISABLED_PROXY_CONFIG;
 import static java.util.Objects.requireNonNull;
 
 import java.net.InetSocketAddress;
 
 import javax.annotation.Nullable;
 
-import com.google.common.base.MoreObjects;
-
 import com.linecorp.armeria.client.ClientFactory;
 
 /**
  * Contains base configuration for proxy related settings used in {@link ClientFactory}.
  */
 public abstract class ProxyConfig {
 
+    ProxyConfig() {
+    }
```

Comment:
Please move down.

Reply:
Sorry, I always seem to forget this :P

Reply:
No worries. We love you, fellow contributor. :heart: